src/app/static/src/js/components/reportLog/runningReportsList.vue

```diff
@@ -61,9 +61,6 @@
             if (this.initialSelectedKey){
                 this.query = this.reports.find(report => report.key === this.initialSelectedKey).name
             }
-        },
-        beforeDestroy() {
-            this.$emit('update:key', "");
```

Comment:
I removed this because I wanted the key to persist in the session if the user switches from the log tab to the run tab and back again.